src/scales/quantitativeScale.ts

```diff
@@ -4,9 +4,10 @@ module Plottable {
   export class QuantitativeScale<D> extends Scale<D, number> {
     protected static _DEFAULT_NUM_TICKS = 10;
     protected _d3Scale: D3.Scale.QuantitativeScale;
-    public _userSetDomainer = false;
-    private _domainer: Domainer = new Domainer();
     private _tickGenerator: Scales.TickGenerators.TickGenerator<D> = (scale: Plottable.QuantitativeScale<D>) => scale.getDefaultTicks();
+    private _padProportion = 0.05;
+    private _paddingExceptions: Utils.Map<any, D>;
+    private _includedValues: Utils.Map<any, D>;
```

Comment:
Possibly, but that wasn't the main thrust of this PR. Thus far the keys are always `Plot`s, but keeping it open is fine.

Maybe there should be some sort of parallel structure for `ExtentsProviders`, but for padding exceptions?